sdk/storagemover/arm-storagemover/CHANGELOG.md

```diff
@@ -1,15 +1,160 @@
+  - Interface AgentUpdateParameters has a new optional parameter properties
+  - Interface Endpoint has a new optional parameter identity
+  - Interface EndpointBaseUpdateParameters has a new optional parameter identity
+  - Interface JobDefinitionUpdateParameters has a new optional parameter properties
+  - Interface JobRun has a new optional parameter properties
+  - Interface Project has a new optional parameter properties
+  - Interface ProjectUpdateParameters has a new optional parameter properties
+  - Interface StorageMover has a new optional parameter properties
+  - Interface StorageMoverUpdateParameters has a new optional parameter properties
+  - Added Type Alias AzureSupportedClouds
+  - Added Type Alias JobType
+  - Added Type Alias ManagedServiceIdentityType
+  - Added Enum AzureClouds
+  - Added Enum KnownJobType
+  - Added Enum KnownManagedServiceIdentityType
+  - Added Enum KnownVersions
+  - Enum KnownEndpointType has a new value AzureMultiCloudConnector
+  - Enum KnownEndpointType has a new value AzureStorageNfsFileShare
 
 ### Breaking Changes
+  - Removed operation Agents.beginDelete
+  - Removed operation Agents.beginDeleteAndWait
+  - Removed operation Endpoints.beginDelete
+  - Removed operation Endpoints.beginDeleteAndWait
+  - Removed operation JobDefinitions.beginDelete
+  - Removed operation JobDefinitions.beginDeleteAndWait
+  - Removed operation Projects.beginDelete
+  - Removed operation Projects.beginDeleteAndWait
+  - Removed operation StorageMovers.beginDelete
+  - Removed operation StorageMovers.beginDeleteAndWait
+  - Operation Agents.createOrUpdate has a new signature
+  - Operation Agents.get has a new signature
+  - Operation Agents.update has a new signature
+  - Operation Endpoints.createOrUpdate has a new signature
+  - Operation Endpoints.get has a new signature
+  - Operation Endpoints.update has a new signature
+  - Operation JobDefinitions.createOrUpdate has a new signature
+  - Operation JobDefinitions.get has a new signature
+  - Operation JobDefinitions.update has a new signature
+  - Operation Projects.update has a new signature
+  - Removed Interface AgentList
+  - Removed Interface EndpointList
+  - Removed Interface JobDefinitionList
+  - Removed Interface JobRunList
+  - Removed Interface ProjectList
+  - Removed Interface StorageMoverList
+  - Removed Interface UploadLimit
+  - Interface Agent has a new required parameter properties
+  - Interface JobDefinition has a new required parameter properties
+  - Type of parameter type of interface Credentials is changed from "AzureKeyVaultSmb" to CredentialType
+  - Type of parameter endpointType of interface EndpointBaseProperties is changed from "AzureStorageBlobContainer" | "NfsMount" | "AzureStorageSmbFileShare" | "SmbMount" to EndpointType
+  - Type of parameter endpointType of interface EndpointBaseUpdateProperties is changed from "AzureStorageBlobContainer" | "NfsMount" | "AzureStorageSmbFileShare" | "SmbMount" to EndpointType
```

Comment:
With above breakings could you share the typespec and swagger with me? I guess some changes in enum?